### PR TITLE
Support Windows Java SE runtime

### PR DESCRIPTION
**Pull request by Flanker32**
Support Windows Java SE runtime, remove old workaround(`web.config`)

### CODE REVIEW

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java [hunk 1]
```diff
@@ -77,21 +77,15 @@ public void publish(DeployTarget deployTarget) throws IOException, MojoExecution
...
         final File parent = new File(stagingDirectoryPath);
         parent.mkdirs();
-
-        if (StringUtils.isNotEmpty(linuxRuntime)) {
-            Files.copy(jar, new File(parent, DEFAULT_LINUX_JAR_NAME));
-        } else {
-            Files.copy(jar, new File(parent, jar.getName()));
-            WebAppUtils.generateWebConfigFile(deployTarget, jar.getName(), stagingDirectoryPath, log);
-        }
+        Files.copy(jar, new File(parent, DEFAULT_LINUX_JAR_NAME));
```
Comment (andxu) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java]: DEFAULT_LINUX_JAR_NAME should be renamed to DEFAULT_JAR_NAME ?

Comment (andxu) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java]: DEFAULT_LINUX_JAR_NAME and DEFAULT_JAVA_SE_JAR_NAME are same?

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/RuntimeSetting.java [hunk 2]
```diff
@@ -27,6 +27,10 @@
 
     public static final String RUNTIME_CONFIG_REFERENCE = "https://aka.ms/maven_webapp_runtime";
 
+    static {
+        WebContainer.fromString("java 11"); // Add Java 11 Enum as Fluent SDK had not added it yet
```
Comment (andxu) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/RuntimeSetting.java]: JAVA_11_STRING ?

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java (hunk 1 above)
Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java]: yes, we need rename the executable jar file to `app.jar`, both for windows and linux

Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java]: Renamed to `DEFAULT_APP_SERVICE_JAR_NAME`

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/RuntimeSetting.java (hunk 2 above)
Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/RuntimeSetting.java]: Fixed

**Pull request merged**